scripts/deploy_preview.sh

```diff
@@ -28,14 +28,29 @@ export AWS_ACCESS_KEY_ID=$(echo $sts_output | jq -r '.Credentials''.AccessKeyId'
 export AWS_SECRET_ACCESS_KEY=$(echo $sts_output | jq -r '.Credentials''.SecretAccessKey');\
 export AWS_SESSION_TOKEN=$(echo $sts_output | jq -r '.Credentials''.SessionToken');
 
+# If recreate option is give first delete the existing resources and create them later.
+if [[ -n "${RECREATE-}" ]]
+then
+  kubectl delete ns $NAMESPACE || true
+  mongosh "mongodb+srv://$DB_USERNAME:$DB_PASSWORD@$DB_URL/$DBNAME?retryWrites=true&minPoolSize=1&maxPoolSize=10&maxIdleTimeMS=900000&authSource=admin" --eval 'db.dropDatabase()'
+  ACCESS_POINT_ID=$(aws efs describe-access-points --file-system-id "$DP_EFS_ID" | jq -r '.AccessPoints[] | select(.Name=="'"$PULL_REQUEST_NUMBER"'") | .AccessPointId')
+  aws efs delete-access-point --access-point-id $ACCESS_POINT_ID
+fi
+
+## Use DP-EFS and create ACCESS_POINT
+ACCESS_POINT=$(aws efs create-access-point --file-system-id $DP_EFS_ID --tags Key=Name,Value=$PULL_REQUEST_NUMBER)
+ACCESS_POINT_ID=$(echo $ACCESS_POINT | jq '.AccessPointId' | tr -d '"')
```

Comment:
```suggestion
ACCESS_POINT_ID=$(echo $ACCESS_POINT | jq -r '.AccessPointId')
```

![shot-2023-08-03-05-47-34](https://github.com/appsmithorg/appsmith/assets/120119/1f8be0dc-e59f-4226-b26d-b5966ac19ab4)